core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/config/AddStoreCommand.java

```diff
@@ -61,18 +65,27 @@
 
 	@Override
 	public boolean prepare(
-			OperationParams params ) {
-		super.prepare(params);
-
-		Properties existingProps = getGeoWaveConfigProperties(params);
+			final OperationParams params ) {
 
 		// Load SPI options for the given type into pluginOptions.
 		if (storeType != null) {
+			if (storeType.equals("memory")) {
```

Comment:
hmm, interesting, i wonder when this was added and if it works in any capacity?  I don't think it is a working use case really. and should probably be dropped.

Reply:
Keep or leave?

Reply:
delete